packages/astro-uxds/_content/migration.md

```diff
@@ -0,0 +1,599 @@
+| --tree-hover-text-color                        | Additional parts may be added in future 7.X releases to allow this level of customization   |
+| --tree-selected-border-color                   | Additional parts may be added in future 7.X releases to allow this level of customization   |
+| --tree-selected-accent-color                   | Additional parts may be added in future 7.X releases to allow this level of customization   |
+| --tree-expanded-border-color                   | Additional parts may be added in future 7.X releases to allow this level of customization   |
+
+## Angular
+
+### Drop Support for <= v11
+
+**Likelihood of Impact: Medium**
+Our Angular wrapper has dropped support for Angular versions <= 11. This aligns with [Angular's own support policy](https://angular.io/guide/releases).
+
+**Resolution:** It is recommended that you upgrade your application to Angular v12+.
+
+## Clock
+
+### Host Style Changes
+
+**Likelihood of Impact: Low**
+
+Previously it was possible to change the `color` by targeting the `rux-clock` element instead of using the available CSS Shadow Parts.
+
+```
+rux-clock {
+   color: red;
+}
+```
+
+These styles have now been properly moved to the shadow dom and require the use of CSS Shadow Parts.
+
+```
+rux-clock::part(container) {
+   color:red;
+}
+```
+
+**Resolution:** Search your application's CSS for anything that may be targeting `rux-clock` directly and replace with the appropriate CSS Shadow Part.
+
+Clock's default `display` property has been changed from `flex` to `inline-block`.
+
+**Resolution:** If you were relying on the element to be `display: flex`, you can change the `display` yourself on the root.
+
+```
+rux-clock {
+  display: flex;
+}
+```
+
+## Checkbox Group
+
+### Host Style Changes
+
+**Likelihood of Impact: Low**
+
+Previously it was possible to change the following CSS properties by targeting the `rux-checkbox-group` element instead of using the available CSS Shadow Parts: `color`, `font-family`, `font-size`, `font-weight`, `line-height`, `letter-spacing`.
+
+```
+rux-checkbox-group {
+   color: red;
+   font-family: Arial;
+   font-size: 2rem;
+   font-weight: 800;
+   line-height: 1;
+   letter-spacing: 0.3em;
+}
+```
+
+These styles have now been properly moved to the shadow dom and require the use of CSS Shadow Parts.
+
+```
+rux-checkbox-group::part(container) {
+   color: red;
+   font-family: Arial;
+   font-size: 2rem;
+   font-weight: 800;
+   line-height: 1;
+   letter-spacing: 0.3em;
+}
+```
+
+**Resolution:** Search your application's CSS for anything that may be targeting `rux-checkbox-group` directly and replace with the appropriate CSS Shadow Part.
+
+## Classification Marking
+
+**Likelihood of Impact: Low**
+
+The deprecated `footer-banner` CSS Shadow Part has been removed
+
+**Resolution:** Search your application's CSS for anything targeting `::part(footer-banner)` and replace with `::part(footer)`
+
+## Dialog
```

Comment:
Need to add the prop name changes of `modal-title` and `modal-message` to `header` and `message`